scripts/mdlint.sh

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env bash
+
+# set -e
+
+# use markdownlint-cli to check for markdown files
+docker run -v ./book:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest '**/*.md' --ignore node_modules
```

Comment:
I _think_ it might be safer to add a `--rm` here just in case the container fails to exit cleanly?

```suggestion
docker run --rm -v ./book:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest '**/*.md' --ignore node_modules
```